src/ToDoInput.jsx

```diff
@@ -0,0 +1,13 @@
+import ToDoList from './ToDoList';
+
+export default function ToDo({ toDos, removeToDoHandler, appendToDoHandler }) {
+  return (
+    <>
+      <div>
+        <h1>To-do</h1>
+        <input type='text' placeholder='할 일을 입력해 주세요.' />
```

Comment:
속성들은 double quoto(`"`)로 써주는게 좋을 것 같아요
```suggestion
        <input type="text" placeholder="할 일을 입력해 주세요." />
```

Reply:
넵!